recipes/watchdog-gevent/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "watchdog-gevent" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/watchdog_gevent-{{ version }}.tar.gz
+  sha256: d19f1276a728dfb3ae3f1c8ada6e791f1726fd4336dff18b4ea35ed3ade35c6a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - gevent >=1.1
+    - pip
```

Comment:
You probably only need `pip` and `python` in this section.

Reply:
Thanks.